integration_test/external_tests/pages/settings_page.dart

```diff
@@ -31,6 +31,18 @@ AppiumBy skipButtonLocator = const AppiumBy.accessibilityId("SKIP");
 AppiumBy alreadyLinkedHeaderLocator =
     const AppiumBy.accessibilityId("Already linked");
 
+// Convert
+
+AppiumBy newAccountLocator =  const AppiumBy.xpath(
+    '//android.widget.ImageView[@content-desc="New Make a new account with addresses you can use to collect or receive NFTs on Ethereum, Feral File, and Tezos. "]'
```

Comment:
@ppupha 
Please use contains instead of all text. If designer change the text, we will have problem in UI mapping

Reply:
I think I created this locator, please search it.

Reply:
This is relate to another test. I removed it. I will make notes for next time.